api/app/change_log/tests/test_change_log.py

```diff
@@ -0,0 +1,71 @@
+from django.test import TestCase
+from django.utils import timezone
+from freezegun import freeze_time
+
+from change_log.tests.models import TestModel
+
+
```

Comment:
General remark, consider adding a specific implementation of a `ChangeTracker` for the `TestModel` (basically make `BaseChangeTracker` raise a `NotImplementedError` when `changed_data` is called and copy the current `changed_data` implementation to the `TestModel` change tracker).

Reply:
Will do this for the Category (already working on that in a separate branch)